code/modules/clothing/suits/labcoat.dm

```diff
@@ -58,8 +58,8 @@
 
 /obj/item/clothing/suit/storage/toggle/labcoat/science
 	name = "moebius scientist labcoat"
-	desc = "A suit that protects against minor chemical spills. Has a purple stripe on the shoulder."
-	icon_state = "labcoat_tox_open"
+	desc = "A suit that protects against minor chemical spills. This one is used by scientists."
+	icon_state = "labcoat_tox" //makes it start closed
```

Comment:
Why though? All other labcoats start open.

Reply:
It started closed previously. I assumed that was intentional but the icons were fucked

Reply:
All lab coats should start open, it's intentional if it starts open

Reply:
done